backstop/assets/favicon.svg

```diff
@@ -0,0 +1,11 @@
+
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+  <style>
+    path { fill: #008BE1; }
+
+    @media (prefers-color-scheme: dark) {
+      path { fill: #A5D7F5; } 
+    }
+  </style>
+  <path d="M8,1.0002a1.00413,1.00413,0,0,1,.51376.13829L13.52436,4.119A.95115.95115,0,0,1,14,4.93885v6.0975a1.00386,1.00386,0,0,1-.48626.85077l-5,2.9744a1.02348,1.02348,0,0,1-1.02748,0l-4.98453-2.965A1.03179,1.03179,0,0,1,2,11.03635V4.96365a1.004,1.004,0,0,1,.48624-.85086l5-2.97431A1.00412,1.00412,0,0,1,8,1.0002m0-1A2.00636,2.00636,0,0,0,6.975.27905l-5,2.9743A1.9939,1.9939,0,0,0,1,4.96365v6.0727a2.025,2.025,0,0,0,.975,1.7102l5,2.9744a2.02328,2.02328,0,0,0,2.05,0l5-2.9744A1.99376,1.99376,0,0,0,15,11.03635V4.93885a1.95338,1.95338,0,0,0-.975-1.6855l-5-2.9743A2.00636,2.00636,0,0,0,8,.0002ZM8,3.25005a1.0625,1.0625,0,1,1-1.0625,1.0625h0A1.0625,1.0625,0,0,1,8,3.25005Zm3,9.1894a.25006.25006,0,0,1-.1221.2148l-2.725,1.6229a.24974.24974,0,0,1-.255.0004L5.1229,12.639A.24992.24992,0,0,1,5,12.42365v-1.4408a.24991.24991,0,0,1,.3778-.2148L7,11.66667V8.23635l-.8778-.5593A.25.25,0,0,1,6,7.46215V6.01635a.25.25,0,0,1,.3783-.2146l2.5,1.3831A.24988.24988,0,0,1,9,7.39935v4.26732l1.6222-.89862a.25.25,0,0,1,.3778.2148Z"/>
+</svg>
```

Comment:
also, how this .gif file gonna work and where? You are not referencing it anywhere right now

Reply:
Mayank talked me out of using the GIF, will be using [this PNG](https://raw.githubusercontent.com/iTwin/iTwinUI/gh-pages/backstop/assets/logo.png) instead.

Reply:
Right now this PNG is being used by [the homepage](https://itwin.github.io/iTwinUI/) only and it appears when shared on a service that uses Open Graph.
![Screen Shot 2021-07-16 at 2 24 11 PM](https://user-images.githubusercontent.com/849817/125992390-ffc4d3c7-0c28-4041-afbf-3e7a405deed7.png)

It seems like Microsoft Teams does not use Open Graph because I can't get the image to appear in Teams chat.  Any suggestions?
![Screen Shot 2021-07-16 at 2 24 58 PM](https://user-images.githubusercontent.com/849817/125992457-d700d2c3-0a23-4141-9bf3-fae0f9da5317.png)